depsync/depsync.go

```diff
@@ -19,19 +19,30 @@ import (
 const (
 	SyncTopic    = "mdm.DepSync"
 	ConfigBucket = "mdm.DEPConfig"
+	syncDuration = 30 * time.Minute
+	fetchLimit   = 100
 )
 
 type Syncer interface {
-	privateDEPSyncer() bool
+	SyncNow()
+}
+
+type autoassigner struct {
+	filter       string
+	profile_uuid string
```

Comment:
use camel case `profileUUID`

Reply:
d'0h, good catch! autoassigner ought to be camelcased, too.